epoch1/cuda/ee_mumu/SubProcesses/Makefile

```diff
@@ -3,7 +3,8 @@ TOOLSDIR = ../../../../../tools
 INCFLAGS = -I. -I../../src -I$(TOOLSDIR)
 MODELLIB = model_sm
 OPTFLAGS = -O3
-CXXFLAGS = $(OPTFLAGS) -std=c++11 $(INCFLAGS) $(USE_NVTX) -Wall -Wshadow -Wextra -fopenmp $(MGONGPU_CONFIG)
+OMPFLAGS?= -fopenmp
```

Comment:
Why do you want to make this optional?

Reply:
It's a workaround for clang. Its compiler driver doesn't understand `-fopenmp`, but the C++ backend does.

The details are in the commit message of b419dffdb3428ed0ca72d44767ab25f987ae7cc6

Reply:
Thanks! Sorry I should have said before, I had seen some similar issues in clang, this is how I fixed it, does this help?
https://github.com/valassi/madgraph4gpu/commit/97ac750deb148ba89125ed128bbc50ae62074096

Reply:
Essentially -lomp instead of -lgomp, for runTest linking

Reply:
This seem to be orthogonal. My clang driver really doesn't swallow `-fopenmp`. It says that there's no such parameter.

Reply:
Uh strange. I used the SFT installed clang. This is what I get in my clang branch with just those two changes (plus your merges I think)

```
[avalassi@itscrd03 bash] ~/GPU2020/madgraph4gpu/epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum> make
/usr/local/cuda-11.1/bin/nvcc  -O3 -lineinfo -std=c++14 -I. -I../../src -I../../../../../tools -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math  -c gcheck.cu -o gcheck.o
/usr/local/cuda-11.1/bin/nvcc  -O3 -lineinfo -std=c++14 -I. -I../../src -I../../../../../tools -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math  -c gCPPProcess.cu -o gCPPProcess.o
/usr/local/cuda-11.1/bin/nvcc gcheck.o -o gcheck.exe gCPPProcess.o -arch=compute_70 -L../../lib -lmodel_sm -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand
/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0-62e61/x86_64-centos7/bin/clang++  -O3 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c check.cc -o check.o
/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0-62e61/x86_64-centos7/bin/clang++  -O3 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c CPPProcess.cc -o CPPProcess.o
In file included from CPPProcess.cc:444:
./CPPProcess.h:86:9: warning: private field 'm_numiterations' is not used [-Wunused-private-field]
    int m_numiterations;
        ^
1 warning generated.
/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0-62e61/x86_64-centos7/bin/clang++ check.o -o check.exe CPPProcess.o  -O3 -std=c++11 -I. -I../../src -I../../../../../tools -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -ldl -pthread -L../../lib -lmodel_sm -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand
/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0-62e61/x86_64-centos7/bin/clang++  -O3 -std=c++11 -I. -I../../src -I../../../../../tools -I../../../../../tools/googletest/googletest/include/ -DUSE_NVTX -Wall -Wshadow -Wextra -fopenmp  -I/usr/local/cuda-11.1/include/ -c runTest.cc -o runTest.o
ln -sf runTest.cc runTest_tmp.cu
/usr/local/cuda-11.1/bin/nvcc -o runTest.exe CPPProcess.o runTest.o gCPPProcess.o runTest_tmp.cu  -O3 -lineinfo -std=c++14 -I. -I../../src -I../../../../../tools -I../../../../../tools/googletest/googletest/include/ -I/usr/local/cuda-11.1/include/ -DUSE_NVTX -arch=compute_70 -use_fast_math  -ldl -L../../lib -lmodel_sm -L../../../../../tools/googletest/build/lib// -lgtest -lgtest_main -L/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0-62e61/x86_64-centos7/lib -L/usr/local/cuda-11.1/lib64/ -lcuda -lcurand -lcuda -lomp
unlink runTest_tmp.cu
```

This is from `/cvmfs/sft.cern.ch/lcg/releases/clang/10.0.0/x86_64-centos7/setup.sh`